crates/iota-types/src/stardust/output/alias.rs

```diff
@@ -201,4 +201,14 @@ impl AliasOutput {
 
         Ok(move_alias_output_object)
     }
+
+    /// Create a `AliasOutput` from BCS bytes.
+    pub fn from_bcs_bytes(content: &[u8]) -> Result<Self, bcs::Error> {
+        bcs::from_bytes(content)
+    }
+
+    /// Serialize a `AliasOutput` as a `Vec<u8>` of BCS.
+    pub fn to_bcs_bytes(&self) -> Vec<u8> {
+        bcs::to_bytes(&self).unwrap()
```

Comment:
Nit: `unwrap` in these functions seems odd as well. You can either `expect` or just propagate the `Result`.